.env

```
@@ -0,0 +1,39 @@
+# Base Endpoint
+endpoint = "http://192.168.1.37:3000"
```

dotenv-linter (3.3.0)

[warning] [LowercaseKey] The endpoint  key should be in uppercase

(LowercaseKey)